src/app/theme/services/baUtil.js

```diff
@@ -27,6 +27,11 @@
       var g = parseInt( hex.slice(3,5), 16 );
       var b = parseInt( hex.slice(5,7), 16 );
       return 'rgba(' + r + ', ' + g + ', ' + b + ', ' + alpha + ')';
+    };
+
+    this.hasAttr = function (elem, attrName) {
+      let attr = $(elem).attr(attrName);
```

Comment:
`let` is not well supported for now unfortunately: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/let
so please use `var`